### PR TITLE
Update Docusaurus and related dependencies

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
## fixes KILTProtocol/ticket#3028

### CODE REVIEW

**Comment by lukeg90** [comment]
Why not simply remove the dead links so the build doesn't fail?

**Review by lukeg90 — COMMENTED** [review]
Why not simply remove the dead links so the build doesn't fail?

**Comment by ChrisChinchilla** [comment]
@ChrisChinchilla Oh I see now. I'm a bit confused by the CI checks, since I'm still getting a dead link error when I try to build locally, but now I see that the PR checks are showing a different error.

Update: I re-ran the checks and now they're all passing except for the AWS preview 🤷

File: src/components/SnippetBlock/index.js [hunk 1]
```diff
@@ -9,7 +9,8 @@ const SnippetBlock = ({
   leadingSpaces = 2,
   ...props
 }) => {
-  const raw = children.split(/\r?\n/)
+  // I don't understand why this is needed…
+  const raw = children.toString().split(/\r?\n/)
```
Comment (lukeg90) [inline src/components/SnippetBlock/index.js:13]: `split` is a string method and must be called on a string. React children can be strings, but more often they will be JSX, which I guess would have to be converted to a string before splitting it like this.

The mystery for me is how it was working before.

**Review by lukeg90 — COMMENTED** [review]
Nice work! That was no trivial task 🙂

File: src/components/SnippetBlock/index.js (hunk 1 above)
Comment (lukeg90) [reply src/components/SnippetBlock/index.js:13]: btw when copying and pasting code you don't understand, it's best to include a link to the source

**Review by lukeg90 — APPROVED** [review]

File: src/components/SnippetBlock/index.js (hunk 1 above)
Comment (ChrisChinchilla) [reply src/components/SnippetBlock/index.js:13]: @lukeg90 I didn't copy and paste, this came from IDE errors, and so I cast it, but much like you I didn't understand why it worked before, so basically I had the same thought process as you but forgot to remove my thought process 😆

**Pull request merged**